lms/djangoapps/instructor/views/api.py

```diff
@@ -1644,12 +1644,10 @@ def calculate_grades_csv(request, course_id):
     try:
         instructor_task.api.submit_calculate_grades_csv(request, course_key)
         success_status = _("Your grade report is being generated! You can view the status of the generation task in the 'Pending Instructor Tasks' section.")
-        return JsonResponse({"status": success_status})
+        return JsonResponse({"status": success_status}, status=202)
```

Comment:
Cool, thanks for the clarification. I'll admit I don't know ORA super well; my job is to run through prs and flag general issues and questions.